modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFileEntryLocalServiceTest.java

```diff
@@ -464,6 +484,42 @@ public void testCheckinFileEntryDeletesPWC() throws Exception {
 				DLFileEntryConstants.PRIVATE_WORKING_COPY_VERSION));
 	}
 
+	@Test
+	public void testCheckingInFileEntryRefreshesStoreUUID() throws Exception {
```

Comment:
I would change the name of the test.
* First, I would use `Check` instead of `Checking`, as the other tests use
* Second, I would like to show that both the _CheckOut_ and the _CheckIn_ actions are being tested.

Final name could be: `testCheckoutAndCheckinFileEntryRefreshesStoreUUID`

Reply:
I suppose that instead of `Refreshes` it should be `Refresh`:  `testCheckoutAndCheckinFileEntryRefreshStoreUUID`

Reply:
I've changed the method name ("refresh" + "Checkin"). I haven't included "checkout" in the name as every checking necessarily implies a checkout.

Reply:
I've squashed the changes into the "Add tests" commit.